woocommerce/woocommerce
# Tasks chaining side panel

### Changes proposed in this Pull Request:

Adding another side-bar panel triggered by clicking the "Finish setup" button on the Activity panel. This will display on all non-Homescreen pages where the activity panel appears. 

Closes #32152  .

### How to test the changes in this Pull Request:

1. Checkout branch on fresh site. 
2. Perform the follow steps for **each iteration of the task list (control / experiment 1 / experiment 2)**. In order to toggle those experiments, use the WCAdmin Test Helper Plugin after opting into Marketing.
3.  Go through a few of the tasks on the Homescreen to ensure they function as expected, including the first one that triggers the OBW. You don't necessarily need to complete them, so that you can continue to test. Ensure that those that don't navigate to a non-reactified page don't have a page refresh.
4. Go to another reactified page that is not the Homescreen, that shows the Activity Panel, such as "_Customers_." 
5. Click the "Finish Setup" button, and click on the same tasks to ensure that they function as expected. For most tasks this still should not trigger a page refresh, same as Homescreen.
6. Go to an embed page such as "_Orders_." Click the "Finish Setup" button and ensure that the task list appears as it should. 
7. Trigger the tasks from here as well, and ensure that they work. This time most tasks will direct to the correct task on the Homescreen, and will have a page refresh. 
8. Switch to any untested experiment (if you haven't finished) and return to step #3.

### Other information:



### FOR PR REVIEWER ONLY:
